src/utils/fetchTokenDetails.ts

```diff
@@ -0,0 +1,14 @@
+import { ChainId } from '@liquality/cryptoassets';
+
+export const CHAINS_WITH_FETCH_TOKEN_DETAILS = [
+  // EVM
+  { chainId: ChainId.Ethereum, label: 'Ethereum (ETH)' },
+  { chainId: ChainId.Rootstock, label: 'Rootstock (RSK)' },
+  { chainId: ChainId.BinanceSmartChain, label: 'Binance Smart Chain (BSC)' },
```

Comment:
what's the label for?

Reply:
It's part of [this PR.](https://github.com/liquality/wallet/pull/851)
https://github.com/liquality/wallet/blob/06b9d15931bf3b88a1775d70dd27ac6334fcc637/src/views/CustomToken.vue#L30